<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MqttClient</class>
 <widget class="QDialog" name="MqttClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabConnection">
    <attribute name="title">
     <string>Connection</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbHost">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edHost">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="edPort">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>1883</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbUsername">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbPassword">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edUsername">
        <property name="text">
         <string>ssp</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edPassword">
        <property name="text">
         <string>pwd</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>113</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Connect</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDisconnect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>113</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Disconnect</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabMqtt">
    <attribute name="title">
     <string>MQTT Data</string>
    </attribute>
    <widget class="QLabel" name="lbReceived">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>401</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>....last data received go here....</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edToSend">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSendMessage">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edToSend</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSendMessage</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
